x-pack/plugins/actions/server/lib/action_executor.ts

```diff
@@ -486,14 +526,17 @@ function validateAction(
         secrets,
       });
     }
+    if (requeueInvalidTasksConfig?.enabled) {
+      validateRawAction({ isPreconfigured, rawAction });
+    }
 
     return { validatedParams, validatedConfig, validatedSecrets };
   } catch (err) {
     throw new ActionExecutionError(err.message, ActionExecutionErrorReason.Validation, {
       actionId,
       status: 'error',
       message: err.message,
-      retry: false,
+      retry: (taskInfo?.requeueInvalidTask?.attempts || 0) > 0,
```

Comment:
This naming is also confusing to me because I thought this was the config 🙈  Instead of `requeueInvalidTask` could we call it something like `numSkippedRuns`?

Reply:
done :)